.github/workflows/lint.yml

```diff
@@ -45,8 +45,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install \
-            "rstcheck" \
-        ;
+        python -m pip install "sphinx"
+        python -m pip install "rstcheck>=6.0"
```

Comment:
Why not

```suggestion
        python -m pip install \
            "rstcheck>=6.0" \
            "sphinx" \
        ;   
```

Reply:
Somehow this didn't work for me when I tried it. It threw up an error. Or maybe I did something wrong...